src/Uno.UI.Tests/Windows_UI_Xaml_Data/xBindTests/Given_xBind_Functions.cs

```diff
@@ -9,61 +9,77 @@ public class Given_xBind_Functions
 		[TestMethod]
 		public void When_Initial_Value()
 		{
-			var SUT = new Functions_Control();
-
-			Assert.AreEqual(string.Empty, SUT._InstanceProperty.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticProperty.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticPrivateProperty.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticPrivateReadonlyField.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceDP.Text);
-			Assert.AreEqual(string.Empty, SUT._InnerProperty.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticClass_PublicStaticProperty.Text);
-			Assert.AreEqual(string.Empty, SUT.StaticPropertyDeepNestingOnNonStaticClass.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_OneParam.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_OneParam_DP_Update_OneTime.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_OneParam_DP_Update_OneWay.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_TwoParam.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_TwoParam_WithConstant.Text);
-			Assert.AreEqual(string.Empty, SUT._System_Function.Text);
-			Assert.AreEqual(string.Empty, SUT._System_Function_with_Quote.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_Parameterless.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticFunction_Parameterless.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_Boolean_False.Text);
-			Assert.AreEqual(string.Empty, SUT._InstanceFunction_Boolean_True.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticClass_PublicConstField.Text);
-			Assert.AreEqual(string.Empty, SUT._StaticPrivateConstField.Text);
-			Assert.AreEqual(string.Empty, SUT._InnerConstField.Text);
-
-
-			SUT.ForceLoaded();
+			var oldFeatureValue = FeatureConfiguration.BindingExpression.HandleSetTargetValueExceptions;
+			try
+			{
```

Comment:
We probably should set this for all tests n this class with `[TestInitialize]` and `[TestCleanup]`

Reply:
What do you mean by not working? Is it going to fail the tests as there will be an exception that should not be present?

Reply:
@jeromelaban Sent you details. For now I think this test is enough, but let me know if you think otherwise.